src/sharedHelpers/uploadObservation.js

```diff
@@ -57,24 +64,8 @@ const uploadEvidence = async (
     return response;
   };
 
-  const responses = await Promise.all( evidence.map( item => {
-    let currentEvidence = item;
-
-    if ( currentEvidence.photo ) {
```

Comment:
This piece was removed, and I don't see it replaced with a block in another file.
So, just to double-check, this can be removed without replacement?

Reply:
yeah, I spent a while puzzling about this and I think it's fine & cleaner code to remove. it's a piece of awkward code where we were first cleaning the null values, but then we were immediately mapping via `mapPhotoForUpload` so we were overwriting any null values we just cleaned anyway.

i'm going to merge, and will note that if we run into issues testing uploads with photos in our internal build this is a great place to look for bugs.